mitodl/ol-keycloak
# Rename Open to Learn

### What are the relevant tickets?




Part of https://github.com/mitodl/hq/issues/5154

### Description (What does it do?)

Updates usages of the logo and any textual usages of the previous product name to "Learn" / "MIT Learn".

### Screenshots (if appropriate):
![Screenshot 2024-08-08 at 16-50-46 MIT Learn](https://github.com/user-attachments/assets/b3a64a56-35dc-45d6-95f8-7541fd036c7a)
![NpCBNu7SKY5GSp42outP3B](https://github.com/user-attachments/assets/cf31f70e-1087-474a-9dfb-8ae0f439ad2c)

### How can this be tested?

Verify the password reset / confirmation emails use the new logo and that the name is correct in areas where there is a hardcoded string. NOTE: some piece of the UI depend on the display name of the realm. You should set this to "MIT Learn" to match.
